detectable/src/main/java/com/synopsys/integration/detectable/detectables/git/parse/GitFileParser.java

```diff
@@ -0,0 +1,97 @@
+package com.synopsys.integration.detectable.detectables.git.parse;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.LoggerFactory;
+
+import com.synopsys.integration.detectable.detectables.git.model.GitConfigElement;
+import com.synopsys.integration.log.IntLogger;
+import com.synopsys.integration.log.Slf4jIntLogger;
+
+public class GitFileParser {
```

Comment:
Was executing git commands considered instead of parsing?

Reply:
I considered it and it was vetoed